libr/bin/bin.c

```diff
@@ -377,6 +378,8 @@ R_API bool r_bin_open_io(RBin *bin, RBinOptions *opt) {
 			return false;
 		}
 		ut64 seekaddr = is_debugger? opt->baseaddr: opt->loadaddr;
+		r_buf_seek (buf, seekaddr, 1);
+		buf->base_priv = seekaddr;
```

Comment:
why? i think fat bins are a "slice" of a buffer. Why do you set `base_priv`?

Reply:
how should i do that with the new rbuffer api?